.travis.yml

```diff
@@ -9,7 +9,7 @@ env:
  global:
    - PRE_INSTALL_HOOK="cd /home/opam/opam-repository && git pull origin master && opam update -u -y"
    - POST_INSTALL_HOOK="sh ./.travis-ci.sh"
-   - PINS="functoria.dev:--dev functoria-runtime.dev:--dev mirage.dev:. mirage-types.dev:. mirage-types-lwt.dev:. mirage-runtime.dev:."
+   - PINS="functoria.dev:--dev functoria-runtime.dev:--dev mirage.3.1.0:. mirage-types.3.1.0:. mirage-types-lwt.3.1.0:. mirage-runtime.3.1.0:."
```

Comment:
We just need to change it after every release. There's no point putting an upper bound on if it's dev, as that is effectively not an upper bound.  The idea is that the upper bound in the `opam` file is 4.0, so our pinned version here needs to be less than that. We could make travis pin to mirage.3.99...